### ISSUE TITLE
Current version of instruction for js-controller update is not shown

### ISSUE THREAD

**Issue by Feuer-sturm** [ioBroker/ioBroker.admin#1311]
**Describe the bug**  
If you click on the button to show in the host settings the instructions how to update js-controller an older version of the instruction will be shown:
![grafik](https://user-images.githubusercontent.com/25422320/152705171-d9f8ae82-c775-453d-b40f-b15898c8b349.png)

**To Reproduce**  
1. Go to Host settings
2. Click on the button to show the instruction how to update js-controller

**Expected behavior**  
The expectation is that the current text of this page https://github.com/ioBroker/ioBroker.docs/blob/master/admin/de/controller-upgrade.md will be shown in the update instruction dialog window

**Versions:**  
- Admin 5.2.3
- js-controller 4.0.3

**Comment by Feuer-sturm** [comment]
@oweitman 
Wobei hier auch die Ursache sein kann, das gar nicht die aktuelle Version von dieser Seite angezeigt wird https://github.com/ioBroker/ioBroker.docs/blob/master/admin/de/controller-upgrade.md angezeigt wird, sondern irgendwas altes. Vielleicht ist dort auch in der alten Version der Fehler. Wird man vermutlich erst dann prüfen können, wenn die korrekte Version wieder im Admin angezeigt wird.

**Comment by Feuer-sturm** [comment]
@oweitman Meine Antwort sollte eine mögliche Erklärung sein,  warum kein Leerzeichen vor dem Bindestrich beim dem Linux Kommando ist.  In der Github Seite, aus welcher die Dokumentation/ der Inh Inhalt vom Screenshot stammt ist das Leerzeichen vorhanden.

**Comment by GermanBluefox** [comment]
Changed in 5.3.0

**Comment by Feuer-sturm** [comment]
Issue is fixed with Admin 5.3.0

Unix command is shown correctly as well
![grafik](https://user-images.githubusercontent.com/25422320/153483201-a5a0265c-579c-4344-835b-34ea14954659.png)

**Comment by Feuer-sturm** [comment]
With current admin version 5.3.1 the old version of the instruction "Anweisungen zum Upgrade des js-controllers" is shown once again and not the current one.

**Comment by klein0r** [comment]
> Unix command is shown correctly as well

Is that still correct? What's the difference to ``iob fix`` ?

**Comment by Apollon77** [comment]
iob fix exists since[ a special point in time of the installer](https://github.com/ioBroker/ioBroker/blob/master/CHANGELOG_INSTALLER_LINUX.md#2019-06-29) ... so depending on when fixer/installer was last used it will exist of you need to execute the shell command (in fact iob fix will also only execute this shell command)

**Comment by Apollon77** [comment]
@GermanBluefox Plan was to always load the content dynamicaly from e.g. https://github.com/ioBroker/ioBroker.docs/blob/master/admin/de/controller-upgrade.md ... only if this is not available we should show an "static variant" (ideally dynamicylly added on release of each admin version

**Comment by mcm1957** [comment]
I've encountered the same problem. See https://forum.iobroker.net/topic/53153/update-js-controller-verbesserungsw%C3%BCrdige-anleitung?_=1646680822191

In addtion I would like to inform a "problem" which hit me as a very new user. 

I spent noticeable time to realize what the "1" near the HOST symbol tries to indicate. And I could not realize what the version number at line named identical to my hostname refers to. I assumed thats a version of Raspion OS or so. It took some time that I realized, thats the js-controller version.

the "problem" was taht as all forum topics and at github we talk about js-controller, js-controller upgrade and so an. But the host panel does not mention the name js-controller anywhere.

I would suggest to evaluate whether the wording js-controller / jscontroller-version should be added at the hist panel at some place. For me theres no more problem of course but I assume, that other new users might be confused too. So please read this as a feedback of a new user.

Note: German verion of this feedback is present at above forum thread.

McM

**Comment by Feuer-sturm** [comment]
Admin 6.1.6
With current admin version there is still the old static update instruction visible and not the content from  https://github.com/ioBroker/ioBroker.docs/blob/master/admin/de/controller-upgrade.md

**Comment by Feuer-sturm** [comment]
@GermanBluefox 
Admin 6.1.7
## ok
The current instruction is now shown :-)

## not ok

1. If you are pressing the button "In die Zwischenablage kopieren" there is a little popup "copied" but if I insert the content in a text editor there is nothing from the instructions
2. Would it be better to have a copy icon after each command like you implemented it for single commands (2nd screenshot). If you have only one button the comments for some commands needs to removed. This can be avoid if there is a copy icon for each command
3. Is it possible to "highlight" in some way the commands so that it is easier to differ between text and command?

![grafik](https://user-images.githubusercontent.com/25422320/176499230-0364eca8-c68f-4789-9379-c924d8d773af.png)
![grafik](https://user-images.githubusercontent.com/25422320/176499760-d1a14886-5481-46cf-aa6a-a6b03ea44c35.png)

**Comment by GermanBluefox** [comment]
![image](https://user-images.githubusercontent.com/4582016/176945709-963775e6-9902-48fa-9974-09ede74f0a9e.png)

**Comment by Feuer-sturm** [comment]
@GermanBluefox 
I made my tests with the not working copy pastewith Firefox and Chrome with Admin 6.1.7

Admin 6.1.8
Everything is fine and working :-) Thank you.

@mcm1957 @GermanBluefox 
As a feedback to your suggestion to write some explantion about js-controller and "Host".
I made proposal in a pullrequest https://github.com/ioBroker/ioBroker.docs/pull/390/files to add some explanation to the update instruction.
Perhaps @GermanBluefox has a good idea where in the "Host menu" can be a hint that the shown version is related to the js-controller
